core/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -832,11 +832,19 @@ public int hashCode() {
     public interface Snapshot extends Closeable {
 
         /**
-         * The total number of operations in the translog.
+         * The total estimated number of operations in the snapshot.
          */
         int totalOperations();
 
         /**
+         * The number of operations has been skipped in the snapshot so far.
+         * Unlike {@link #totalOperations()}, this value is updated each time after {@link #next()}) is called.
+         */
+        default int skippedOperations() {
```

Comment:
I personally find `skippedOperations` a bit tricky as we don't really skip operations. Rather we remove operations that have been overridden. Maybe "overridenOperations" and update the java docs to explain what overridden means?

Reply:
I pushed https://github.com/elastic/elasticsearch/pull/27268/commits/5cc33bbc2505fd94eeaa4bd1f4834e55f3d79335